datashare-app/src/main/java/org/icij/datashare/web/TaskResource.java

```diff
@@ -573,6 +602,27 @@ public record ErrorResponse(String message) {}
     public record TaskResponse(String taskId) {}
     public record TasksResponse(List<String> taskIds) {}
 
+    private WebQueryPagination getPagination(Context context) {
+        Set<String> paginationFields = WebQueryPagination.fields();
```

Comment:
This is a computed field and it's constant, we could extract it to avoid re-computing it every time.

Reply:
Done!